s3/src/test/scala/docs/scaladsl/S3SinkSpec.scala

```diff
@@ -286,10 +288,13 @@ class S3SinkSpec extends S3WireMockBase with S3ClientIntegrationSpec with Option
         .withHeader(sseCSourceKeyHeader, new EqualToPattern(sseCSourceKeyHeaderValue))
         .withHeader(requestPayerHeader, new EqualToPattern(requestPayerHeaderValue)))
 
-    // No SSE-C headers required for CompleteMultipartUpload
+    // SSE headers only
     mock.verifyThat(
       postRequestedFor(urlEqualTo(s"/$targetBucketKey?uploadId=$uploadId"))
-        .withHeader(requestPayerHeader, new EqualToPattern(requestPayerHeaderValue)))
+        .withHeader(sseCAlgorithmHeader, new EqualToPattern(sseCAlgorithmHeaderValue))
+        .withHeader(sseCKeyHeader, new EqualToPattern(sseCKeyHeaderValue))
+        .withHeader(sseCKeyHeaderMd5, new EqualToPattern(sseCKeyHeaderMd5Value))
+        .withoutHeader(requestPayerHeader))
```

Comment:
`new EqualToPattern(...)` would looks better with `EqualToPattern(...) `

Reply:
I just followed the same style that is done in the rest of the test